regparser/notice/compiler.py

```diff
@@ -258,6 +261,12 @@ def replace_node_and_subtree(self, node):
         else:
             parent = self.get_parent(node)
 
+        if parent is None:
+            # This is a corner case, where we're trying to add a child to a
+            # parent that should exist.
+            logging.warning('No existing parent for: %s' % node.label_id())
+            parent = self.create_empty_node(get_parent_label(node))
```

Comment:
@grapesmoker : here's the strategy I went with when getting all of the regs to parse before. I *think* this was originally added due to implicit interpretation nodes (the authors adding an interpretation of a paragraph without mentioning its parents). In any event, it may resolve your reg Z problem if it was that kind of interpretation issue, and give you a useful warning if not.